### ISSUE TITLE
Search store guide

### ISSUE THREAD

**Issue by mthuret** [algolia/vue-instantsearch#309]
Maybe I missed it on the current document but it would be great to have a guide detailing the structure of the `searchStore` like we do for React InstantSearch: https://community.algolia.com/react-instantsearch/guide/Search_state.html

I found myself looking for this kind of information, and we had user looking for it too on RIS.

What do you think @rayrutjes?

**Comment by rayrutjes** [comment]
Yes we could document that more indeed.
For now, what we document is the different ways of using the search store: https://community.algolia.com/vue-instantsearch/getting-started/search-store.html

But I think documenting the API of it could be a great addition.

**Comment by pixelastic** [comment]
I'm adding my voice to it as well. I found some answers by reading the source code of the store and the components using it, but I am still confused as to how to use it. Should I use the internal helper methods, or change the store values?

**Comment by mandersondesign** [comment]
👍 This is hard to parse right now. A lot of the same things that we could do with the plain javascript version of instantsearch, I think are possible with the vue implementation. but we have no direct way to actually figure out how that works...

**Comment by Haroenv** [comment]
search store is going away in the next version of Vue InstantSearch (planned for the coming months). Instead we will be reusing infra from the vanilla InstantSearch. For now, feel free to ask any questions you have.

**Comment by Haroenv** [comment]
This issue is no longer relevant with v2 #416, since the concept of search store goes away
